doc/articles/uno-development/guidelines-for-updating-dependencies.md

```diff
@@ -0,0 +1,51 @@
+# Guidelines for updating dependencies
```

Comment:
1) Needs to be added to the TOC
2) Change filename to `contributing/guidelines/updating-dependencies.md`